pkg/insights/insightsclient/insightsclient.go

```diff
@@ -128,7 +137,7 @@ func (c *Client) Send(ctx context.Context, endpoint string, source Source) error
 	if req.Header == nil {
 		req.Header = make(http.Header)
 	}
-	req.Header.Set("User-Agent", fmt.Sprintf("insights-operator/%s cluster/%s", version.Get().GitCommit, cv.Spec.ClusterID))
+	req.Header.Set("User-Agent", fmt.Sprintf("insights-operator/%s cluster/%s version/%s", version.Get().GitCommit, cv.Spec.ClusterID, getCurrentVersion(cv)))
```

Comment:
@jhjaggars you mentioned some changed needed on ingress side before we can add additional fields to user-agent header. Mind adding more details here?

Reply:
Yes, the code that parses the user-agent will need to be updated to support the additional field in a backward compatible way.  Have you considered putting the version in the `insights-operator/$VERSION` spot already?